Bottom: 289d8bd380a87990cd1cc50667e6d395495c6d7e
Top:    0172c89cad773f0ffa018794dc93425c51f82f23
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 14:25:29 +0800

Refresh of autofs-5.0.7-use-LIBS-for-link-libraries.patch

---

diff --git a/Makefile.rules b/Makefile.rules
index d00ba3c..b4fab4c 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -45,7 +45,7 @@ LD        ?= ld
 SOLDFLAGS = -shared
 
 CFLAGS += -D_REENTRANT -D_FILE_OFFSET_BITS=64
-LDFLAGS += -lpthread
+LIBS += -lpthread
 
 ifdef TIRPCLIB
 CFLAGS += -I/usr/include/tirpc
